src/index.ts

```diff
@@ -1,93 +1,155 @@
-import * as core from '@actions/core';
-import * as github from '@actions/github'
-import { LinearClient } from '@linear/sdk'
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import { LinearClient } from "@linear/sdk";
 import { isNil } from "lodash";
 
 const PR_TITLE_UPDATE_KEYWORD = "x";
 
 type PullRequest = {
-    number: number;
-    title: string;
-    head: {
-        ref: string;
-        repo: {
-            name: string;
-            owner: {
-                login: string;
-            }
-        } | null,
-    },
-}
+  number: number;
+  title: string;
+  body: string | null;
+  head: {
+    ref: string;
+    repo: {
+      name: string;
+      owner: {
+        login: string;
+      };
+    } | null;
+  };
+};
 
-export const getIssueIdFromBranchName = (branchName: string): string | null => {
-    const match = branchName.match(/^feature\/([a-z]+\-\d+)\-/);
-    if (isNil(match)) return null;
-    const branchStoryId = match[1];
-    return branchStoryId.toUpperCase();
-}
+export const getLinearIssueIds = (pullRequest: PullRequest) => {
+  const issueIds: string[] = [];
+
+  // From the branch name
+  const branchName = pullRequest.head.ref;
+  let match = branchName.match(/^([a-z]+\-\d+)\-/);
+  if (!isNil(match)) issueIds.push(match[1].toUpperCase());
+
+  // From the PR body
+  const body = isNil(pullRequest.body) ? "" : pullRequest.body;
```

Comment:
`const body = pullRequest.body ?? "";`

Reply:
Done in https://github.com/perdoo/linear-pr-title-action/pull/2/commits/9ff441d9b382e71b3cdee9fb9cfea0ab16d2598d.